src/main/java/io/jenkins/plugins/gitlabserver/servers/GitLabServers.java

```diff
@@ -137,10 +118,10 @@ public synchronized void setServers(@CheckForNull List<? extends GitLabServer> s
      * @param endpoint the endpoint to add.
      * @return {@code true} if the list of endpoints was modified
      */
-    public synchronized boolean addServer(@Nonnull GitLabServer endpoint) {
+    public boolean addServer(@Nonnull GitLabServer endpoint) {
         List<GitLabServer> endpoints = new ArrayList<>(getServers());
         for (GitLabServer ep : endpoints) {
-            if (ep.getServerUrl().equals(endpoint.getServerUrl())) {
+            if (Util.fixNull(ep.getName()).equals(Util.fixNull(endpoint.getName()))) {
```

Comment:
Is the change to `getName()` instead of `getServerUrl()` to allow people to have multiple configurations for the same server?

Reply:
Perhaps provide more details on the following (or link to bug in JIRA with more information) for this:

`Fixed a minor bug of changing param displayName to name in DataboundConstructor of GitLabServer DescriptorImpl`

Reply:
Yeah, I missed this part. PR #4 is all about that.

Reply:
Perhaps provide more details on the following (or link to bug in JIRA with more information) for this:

`Fixed a minor bug of changing param displayName to name in DataboundConstructor of GitLabServer DescriptorImpl`

Reply:
Created an issue: https://issues.jenkins-ci.org/browse/JENKINS-57885